idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/TestShuffler.java

```diff
@@ -194,4 +195,32 @@ private int permutations(final int accum, final Iterator<List<String>> iterator,
             }
         }
     }
+
+    public List<String> alphabeticalAndTuscanOrder(int count, boolean isTuscan) {
+        List<String> classes = new ArrayList<>(classToMethods.keySet());
+        Collections.sort(classes);
+        final List<String> fullTestOrder = new ArrayList<>();
+        if (isTuscan) {
+            int n = classes.size();
+            int[][] res = Tuscan.generateTuscanPermutations(n);
+            List<String> permClasses = new ArrayList<String>();
+            if (n == 3 || n == 5) {
```

Comment:
If you end up simplifying and always have it return a matrix where the last column of each row is 0, then can simplify this conditional.